testnet/testnet-rewards.md

```diff
@@ -69,14 +69,124 @@ Validators earn $JOY allocations in the same way as for bounty contributions: a
 
 All council members earn the same quantity of $tJOY per period, let `TOTAL_COUNCIL_tJOY_REWARD` denote the total $tJOY payout earned by the entire council. This value is controlled by Jsgenesis, and can change over time.
 
-Council Members earn $JOY a bit differently, specifically, the total $JOY amount shared equally among all members depends what is called the _network peformance score,_ given by\
-\
-`(TOTAL_COUNCIL_tJOY_REWARD/tJOY_BUDGET)*(NETWORK_PERFORMANCE_SCORE)^2`\
-\
-where `NETWORK_PERFORMANCE_SCORE`, and is in the interval \[0, 1]. This score, computed by Jsgenesis using public evaluation metrics, attempts to capture how well the network performed. The evaluation metrics will change from one council period to the next.
+Council Members earn $JOY a bit differently, specifically, the total $JOY amount shared equally among all members are reduced by what is called the _network peformance score,_ squared, as shown below:
+
+```
+TOTAL_COUNCIL_JOY_REWARD = JOY_BUDGET * (TOTAL_COUNCIL_tJOY_REWARD/tJOY_BUDGET) * NETWORK_PERFORMANCE_SCORE^2
+```
 
-Council members earn $JOY allocations: a share of the `JOY_BUDGET` equal to what share the earned $tJOY makes up of the `tJOY_BUDGET`.
+where `NETWORK_PERFORMANCE_SCORE`, and is in the interval \[0, 1]. This score, computed by Jsgenesis using public evaluation metrics, attempts to capture how well the network performed. The evaluation metrics will change from one council period to the next.
```

Comment:
Will they really be changing that much from one period to the next?
I thought the idea was that, in this startup phase, they can change, but it makes not so much sense for them to change every week going forward, then you just have KPIs squared and way too much involvement from us and not enough time for the Council to calibrate/understand the things that are going to have the most effect on the network score.

Reply:
Hopefully not, but although the metrics stay the same, the values may not!

The main reason for having this, is that we want to avoid "gaming" the system...